docs: remove beta state of postgresql (#5652)
zitadel/zitadel: docs/docs/support/general.md

---

> title: General

Reviewer: @hifabienne regarding structure:  Can we call this page "Software release cycle"? and move the support states to https://zitadel.com/docs/legal/support-services ? 

But let's fix first the content comments.

Reviewer: Note to self: clarify enterprise supported features in self-hosted and extended sla, the default (included/excluded), and how to become eligible

Author: Yes makes sense

---

## Support States

It's important to note that support may differ depending on the feature, and not all features may be fully supported. 
We always strive to provide the best support possible for our customers and community,
but we may not be able to provide immediate or comprehensive support for all features.
Also the support may differ depending on your contracts. Read more about it on our [Legal Page](/docs/legal/introduction)

### Fully Supported

Fully supported features are those that are guaranteed to work as intended and are fully tested by our team.
If you encounter any issues with a fully supported feature, you can contact us, 

> and we will provide prompt resolution to any issues you may experience.

Author: Should we remove the fully here as well?

> and we will provide prompt resolution to any issues you may experience.

Reviewer: Updated suggestion

### Partially Supported

> Partially supported features are those that we support to a certain extent, 
> but may not have been fully tested or may have some limitations.
> If you encounter issues with a partially supported feature, we will make our best effort to assist you, 
> but we may not be able to provide immediate resolution or comprehensive support.

Reviewer: ```suggestion
Enterprise supported features are those where we provide support only to users eligible for enterprise [support services](/docs/legal/support-services).
These features should be functional for eligible users, but may have some limitations for a broader use.

If you encounter issues with an enterprise supported feature and you are eligible for enterprise support services, we will provide a prompt response to the issues you may experience and make our best effort to assist you to find a resolution.
```

**Partially supported features**

- Database PostgreSQL

### Community Supported

Community supported features are those that have been developed by our community and may not have undergone extensive testing or support from our team. 

> If you encounter issues with a community supported feature, we encourage you to seek help from our community or other online resources, where other users can provide assistance.

Reviewer: ```suggestion
If you encounter issues with a community supported feature, we encourage you to seek help from our community or other online resources, where other users can provide assistance:
```

> Join our [Discrod Chat](https://zitadel.com/chat)

Reviewer: ```suggestion
- Join our [Discord Chat](https://zitadel.com/chat)
- Search [Github Issues](https://github.com/search?q=org%3Azitadel+&type=issues) and report a new issue
- Search [Github Discussions](https://github.com/search?q=org%3Azitadel+&type=discussions) and open a new discussion as question or idea
```

## Software Release Cycle

It's important to note that both Alpha and Beta software can have breaking changes, meaning they are not backward-compatible with previous versions of the software.
Therefore, it's recommended to use caution when using Alpha and Beta software, and to always back up important data before installing or testing new software versions.

> 

Reviewer: ```suggestion

Only features in General Availability will be covered by support services.

We encourage our community to test Alpha and Beta software and provide feedback via our [Discord Chat](https://zitadel.com/chat). 

```

Author: Do you think it is not relevant to state that alpha and beta can have breaking changes?
And also that its recommended to use it with caution?

Reviewer: Yes. I wanted to make an addition. Does it delete something on Line 40?

### Alpha

The Alpha state is our initial testing phase. 
It is available to everyone, but it is not yet complete and may contain bugs and incomplete features.
We recommend that users exercise caution when using Alpha software and avoid using it for critical tasks, as support is limited during this phase.

### Beta

The Beta state comes after the Alpha phase and is a more stable version of the software. 

> It is feature-complete, but may still contain bugs that need to be fixed before release.

Reviewer: ```suggestion
It is feature-complete, but may still contain bugs that need to be fixed before general availability.
```

While it is available to everyone, we recommend that users exercise caution when using Beta software and avoid using it for critical tasks. 
During this phase, support is limited as we focus on testing and bug fixing. 

> 

Reviewer: ```suggestion

### General Available

Generally available features are available to everyone and have the appropriate test coverage to be used for critical tasks.
The software will be backwards-compatible with previous versions, for exceptions we will publish a [technical advisory](https://zitadel.com/docs/support/technical_advisory).
Features in General Availability are not marked explicitly.

```

We always strive to provide the best support possible for our customers and community,
but we may not be able to provide immediate or comprehensive support for all features.
Also the support may differ depending on your contracts. Read more about it on our [Legal page](/docs/legal/)

### Supported

Supported features are those that are guaranteed to work as intended and are fully tested by our team.
If you encounter any issues with a fully supported feature, you can contact us, 
and we will provide prompt resolution to any issues you may experience.

### Enterprise Supported

Enterprise supported features are those where we provide support only to users eligible for enterprise [support services](/docs/legal/support-services).
These features should be functional for eligible users, but may have some limitations for a broader use.

If you encounter issues with an enterprise supported feature and you are eligible for enterprise support services, we will provide a prompt response to the issues you may experience and make our best effort to assist you to find a resolution.

> **Partially supported features**

Reviewer: ```suggestion
**Enterprise supported features**
```

> - Database PostgreSQL

Reviewer: ```suggestion
- Database PostgreSQL
- LDAP Identity Provider
- [Terraform Provider](https://github.com/zitadel/terraform-provider-zitadel)
```
